hypha/apply/funds/forms.py

```diff
@@ -324,6 +324,35 @@ def make_role_reviewer_fields():
     return role_fields
 
 
+class UpdateAuthorForm(ApplicationSubmissionModelForm):
+    author = forms.ModelChoiceField(
+        queryset=User.objects.applicants(),
+        label=_("Applicants"),
+        required=False,
+    )
+    author.widget.attrs.update({"data-placeholder": "Select..."})
```

Comment:
Can we make this `author.widget.attrs.update({"data-placeholder": "Select...", "data-js-choices": ""})`?